error/404.html

```diff
@@ -2,12 +2,17 @@
 <html>
 <head lang="en">
   <meta charset="UTF-8">
-  <title></title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1">
```

Comment:
because I will still make a better template, that's why I put it there.
Example: https://thevidme.club/fgdgdg